core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -5986,6 +5997,18 @@ private void checkLiteral2(String expression, String expected) {
         .ok(expected);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-4724">[CALCITE-4724]
+   * ClickHouseSqlDialect `supportsAliasedValues` should return false. </a>
+   */
+  @Test void testAliasedValueForClickHouse() {
+    final String query = "select 1";
+    final String expected = "SELECT 1";
+    sql(query)
+        .withClickHouse()
+        .ok(expected);
```

Comment:
some tips:  sql(query).withClickHouse().ok(expected);  may be better.

Reply:
adjusted.